extensions/statsd-emitter/src/main/java/io/druid/emitter/statsd/StatsDEmitterConfig.java

```diff
@@ -0,0 +1,100 @@
+package io.druid.emitter.statsd;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.Preconditions;
+
+import java.util.Map;
+
+/**
+ */
+public class StatsDEmitterConfig
+{
+
+  public static enum MetricType {
+    count,
+    timer,
+    guage
+  }
+
+  @JsonProperty
+  final private String hostname;
+  @JsonProperty
+  final private Integer port;
+  @JsonProperty
+  final private String prefix;
+  @JsonProperty
+  final private Map<String, MetricType> metricTypes;
+
+  @JsonCreator
+  public StatsDEmitterConfig(
+      @JsonProperty("hostname") String hostname,
+      @JsonProperty("port") Integer port,
+      @JsonProperty("prefix") String prefix,
+      @JsonProperty("metricTypes") Map<String, MetricType> metricTypes)
+  {
+    Preconditions.checkNotNull(hostname, "StatsD hostname cannot be null.");
```

Comment:
please do use `this.hostname = Preconditions.checkNotNull(hostname, "StatsD hostname cannot be null.");`